pages/index.tsx

```diff
@@ -137,6 +164,22 @@ export const HomePage: NextPage = () => {
       </Stack>
       <Stack spacing="32px" px="112px" py="64px">
         <Button onClick={onDrawerOpen}>View your list</Button>
+        <Select
+          isMulti
+          isSearchable
+          options={allNeighborhoods
+            ?.map((neighborhood) => ({
+              value: neighborhood,
+              label: neighborhood['Neighborhood Name'],
+            }))
```

Comment:
If we make the refactor suggested in the above comment, we could prob also make the option values just neighborhood names instead of the whole `Neighborhood` object.

Reply:
done as well